djangopeople/djangopeople/migrations/0002_load_fixtures.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core.management import call_command
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    call_command('loaddata', 'initial', verbosity=0)
```

Comment:
My official answer is going to be: Either use `RunPython` and `Model.objects.bulk_create()` or `RunSQL` with `INSERT INTO`.

My in-official answer is: do what you need to do to make it work. Comment out this line after you deployed and be done with it.